content/crutchcorn/collections/art-of-accessibility/posts/art-of-a11y-preface/index.md

```diff
@@ -0,0 +1,330 @@
+
+However, you may not be familiar with their work in the accessibility space which is conducted through the WIA group. WIA is comprised of accessibility experts and professionals, organizations representing people with disabilities, policy makers, and more. Together, they publish the **"Web Content Accessibility Guidelines" (WCAG)**, which acts as the de-facto guidelines for what makes a project more or less accessible.
+
+The WCAG guidelines may see revisions and improvements through different version numbers; for example, this book aims to conform to the guidance of WCAG 2.2, but the general goal remains consistent: Help developers and organizations figure out what the best practices are for their apps' UX.
+
+## Different levels of compliance
+
+Humans come in many shapes and sizes, both literally as well as metaphorically. This diversity allows us to have fresh perspectives and ideas that innovate in every facet of life, but it also introduces a high level of permutations in which a user is able to interact with our software.
+
+Similarly, this variation of engagement allows all walks of life to interface with their machines; from newcomers who might hover over every icon to see its meaning, to skilled hackers skipping entire interaction types by never lifting their fingers from their keyboard.
+
+But this variety doesn't come for free. For each unique combination of user behavior, we need to:
+
+- Implement the baseline functionality's needs
+- Test our software against it
+- Edge-case against and fix bugs
+
+And more.
+
+How can we prioritize which features of accessibility we should start on? How can we balance effort against the impact a set of functionality will provide?
+
+Intro: **WCAG Conformance Levels**.
+
+WCAG Conformance Levels are broken into three different categories:
+
+- **A**: The most basic level of WCAG conformance comprising of a set of guidelines to follow. Without these guidelines met, many of your users may face a barrier preventing them from interacting with your work.
+  - EG: Descriptive text for images for non-sighted users, keyboard navigation, and accessible forms.
+- **AA**: Includes all of "A" level rules. May impact the "look and feel" of the site, but provides additional value to all users.
+  - EG: Basic color contrast for visually impaired users, text resizing support, headings level structure.
+- **AAA**: The highest level of WCAG conformance which includes all of "A" and "AA" level rules as well as its own. May furtherly constrain the "look and feel" of the site, but provides the most value to many users.
+  - EG: More rigorous contrast requirements, sign language for live video.
+
+# How can I make my apps accessible?
+
+Making apps more accessible isn't a one-size fits all solution. While some may sell one-click solutions, such as "accessibility overlays", I can confidently say that at best they're snake-oil and at worst they can do more harm than doing nothing at all.
```

Comment:
Reference(s) here may be helpful in abating sceptics of dealing in absolutes for how bad they are.

E.g. [Overlay Factsheet](https://overlayfactsheet.com)